fontbe/src/cmap.rs

```diff
@@ -25,12 +28,10 @@ impl Work<Context, AnyWorkId, Error> for CmapWork {
     }
 
     fn read_access(&self) -> Access<AnyWorkId> {
-        Access::Custom(Arc::new(|id| {
-            matches!(
-                id,
-                AnyWorkId::Fe(FeWorkId::GlyphOrder) | AnyWorkId::Fe(FeWorkId::Glyph(..))
-            )
-        }))
+        Access::Set(HashSet::from([
+            AllOrOne::All(AnyWorkId::Fe(FeWorkId::GlyphOrder)),
```

Comment:
now this is a bit confusing..
Shouldn't this be an AllOrOne::One or `AnyWorkId::Fe(FeWorkId::GlyphOrder).into()`? There is only one hob with FeWorkId::GlyphOrder, not many that you may need to block on _all_?